openedx/adg/common/user_authn_override/constants.py

```diff
@@ -0,0 +1,82 @@
+"""
+Constants related to ADG.
+"""
+
+
+from django.utils.translation import ugettext_lazy as _
+
+# TODO: Move this setting to model
+ADG_BUSINESS_LINES = {
+    ('Tanmiah', _('Tanmiah Food Group')),
+    ('Petromin', _('Petromin')),
+    ('RSI', _('Red Sea International')),
+    ('Dukan', _('Dukan')),
+    ('SAED', _('SAED')),
+    ('BARQ', _('BARQ System')),
+    ('PPC', _('Premium Paints Company')),
+    ('IPD', _('International Project Developers')),
+    ('ADG', _('Al-Dabbagh Group'))
+}
+
+CITIES_OF_KSA = [
```

Comment:
Let's please change the structure to provide better support in future, as they are planning to add more countries in future. Let's rename it to `CITIES` and make it a dictionary with keys of countries and for now use it like `CITIES['KSA']`, in future we can support more countries.
```
CITIES: {
    'KSA': [
        ('Yanbu', _('Yanbu')),
       ('Umm Lajj', _('Umm Lajj')),
       ('Ţurayf', _('Ţurayf')),
        .........
   ]
}
```

Thanks to @zeshanarifios for the idea.